src/03-deals/deals.apib

```diff
@@ -153,11 +153,11 @@ Get details for a single deal.
                 + (object)
                     + id: `e2314517-3cab-4aa9-8471-450e73449041` (string)
                     + type: `quotation` (string)
-            + lost_reason (object)
-                + reason (object)
+            + lost_reason (object, nullable)
+                + reason (object, nullable)
                     + type: lostReason (string)
                     + id: `4e81f3c4-7dca-44cb-b126-6158bec392a2` (string)
-                + remark: `Too expensive` (string)
+                + remark: `Too expensive` (string, nullable)
```

Comment:
Q: can it happen that `lost_reason` is provided, but both or either reason or remark are `null`?

Reply:
Explained by @luisaterrabrito IRL; if the reason is unknown, there is no `lostReason` set.